### ISSUE TITLE
[Bug]: Spelling error

### ISSUE THREAD

**Issue by AnthonyEagle1776** [mfdlabs-grid-development/grid-bot-support#4]
### Check if there is an existing issue or pull request.

- [X] I double checked issues and couldn't find any duplicates.

### This occurs on the latest version of the bot.

- [X] I verified that the bot is running on the latest version.

### Does this bug occur frequently?

- [ ] This bug occurs frequently.

### Description

there's a spelling error, the 'to' after templates should be changed to 'so'

### Images

![BRRGB](https://user-images.githubusercontent.com/22604361/158076978-9ce2f7f7-b591-4402-8313-030327297674.png)

### Steps to Reproduce

1. Execute ;deploy or whatever the prefix is that is set.

**Comment by mfdlabs-ops** [comment]
(Awaiting Assignment)

**Comment by mfdlabs-ops** [comment]
@AnthonyEagle1776, this issue has been assigned to (@nkpetko), development branch is `fix/inconsistent-spelling-in-commands`.

Development continues here.

**Comment by mfdlabs-ops** [comment]
@AnthonyEagle1776 change commited to repository [mfdlabs/grid-bot@fix/inconsistent-spelling-in-commands](https://git.mfdlabs.local/mfdlabs/grid-bot/tree/fix/inconsistent-spelling-in-commands) with hash [fix/inconsistent-spelling-in-commands@76f30e7c49bb8f544a5d8d196a03a21b301da8c4](https://git.mfdlabs.local/mfdlabs/grid-bot/commit/76f30e7c49bb8f544a5d8d196a03a21b301da8c4)

This change will not be integrated into master until CI checks complete and the pull request associated with it does not fail

The pull request associated with this is: [mfdlabs/grid-bot#272](https://git.mfdlabs.local/mfdlabs/grid-bot/pull/272)

Please review the diff below:

```diff
diff --git a/Shared/MFDLabs.GridCommands/Commands/DeploymentId.cs b/Shared/MFDLabs.GridCommands/Commands/DeploymentId.cs
index 0d10c3c..a266847 100644
--- a/Shared/MFDLabs.GridCommands/Commands/DeploymentId.cs
+++ b/Shared/MFDLabs.GridCommands/Commands/DeploymentId.cs
@@ -25,7 +25,10 @@ public async Task Invoke(string[] messageContentArray, SocketMessage message, st
             var currentDirectoryName = Path.GetFileName(currentDirectory);
 
             // Reply with the deployment ID.
-            await message.ReplyAsync($"The deployment ID for this instance is: `{currentDirectoryName}`\nPlease paste this into the `Deployment ID` field in grid-bot-support templates to that the internal team can easily identify this instance.");
+            await message.ReplyAsync(
+                $"The deployment ID for this instance is: `{currentDirectoryName}`\n" + 
+                "Please paste this into the `Deployment ID` field in grid-bot-support templates so that the internal team can easily identify this instance."
+            );
         }
     }
 }
\ No newline at end of file
diff --git a/Shared/MFDLabs.GridCommands/Commands/Hostname.cs b/Shared/MFDLabs.GridCommands/Commands/Hostname.cs
index d842f0c..a0c644d 100644
--- a/Shared/MFDLabs.GridCommands/Commands/Hostname.cs
+++ b/Shared/MFDLabs.GridCommands/Commands/Hostname.cs
@@ -25,7 +25,7 @@ public async Task Invoke(string[] messageContentArray, SocketMessage message, st
                 $"The hostname for this instance is: `{SystemGlobal.GetMachineHost()}`\n" +
                 $"The machine ID for this machine is: `{SystemGlobal.GetMachineId()}`\n" +
                 $"The IP address for this machine is: `{NetworkingGlobal.GetLocalIp()}`\n" +
-                "Please paste this into the `Host Name` field in grid-bot-support templates to that the internal team can easily identify this instance."
+                "Please paste this into the `Host Name` field in grid-bot-support templates so that the internal team can easily identify this instance."
             );
         }
     }
diff --git a/Shared/MFDLabs.GridCommands/Commands/LogFile.cs b/Shared/MFDLabs.GridCommands/Commands/LogFile.cs
index 5d62fec..473c1bf 100644
--- a/Shared/MFDLabs.GridCommands/Commands/LogFile.cs
+++ b/Shared/MFDLabs.GridCommands/Commands/LogFile.cs
@@ -17,7 +17,10 @@ internal sealed class LogFile : IStateSpecificCommandHandler
 
         public async Task Invoke(string[] messageContentArray, SocketMessage message, string originalCommand)
         {
-            await message.ReplyAsync($"The log file name for this instance is: `{SystemLogger.FileName}`\nPlease paste this into the `Log File Name` field in grid-bot-support templates to that the internal team can easily identify this instance's log files.");
+            await message.ReplyAsync(
+                $"The log file name for this instance is: `{SystemLogger.FileName}`\n" + 
+                "Please paste this into the `Log File Name` field in grid-bot-support templates so that the internal team can easily identify this instance's log files."
+            );
         }
     }
 }
\ No newline at end of file
diff --git a/Shared/MFDLabs.GridCommands/SlashCommands/DeploymentId.cs b/Shared/MFDLabs.GridCommands/SlashCommands/DeploymentId.cs
index 6056175..ad5794c 100644
--- a/Shared/MFDLabs.GridCommands/SlashCommands/DeploymentId.cs
+++ b/Shared/MFDLabs.GridCommands/SlashCommands/DeploymentId.cs
@@ -31,7 +31,10 @@ public async Task Invoke(SocketSlashCommand command)
                 var currentDirectoryName = Path.GetFileName(currentDirectory);
 
                 // Reply with the deployment ID.
-                await command.RespondEphemeralAsync($"The deployment ID for this instance is: `{currentDirectoryName}`\nPlease paste this into the `Deployment ID` field in grid-bot-support templates to that the internal team can easily identify this instance.");
+                await command.RespondEphemeralAsync(
+                    $"The deployment ID for this instance is: `{currentDirectoryName}`\n" + 
+                    "Please paste this into the `Deployment ID` field in grid-bot-support templates so that the internal team can easily identify this instance."
+                );
             }
         }
     }
diff --git a/Shared/MFDLabs.GridCommands/SlashCommands/Hostname.cs b/Shared/MFDLabs.GridCommands/SlashCommands/Hostname.cs
index 4b9eed5..d8819be 100644
--- a/Shared/MFDLabs.GridCommands/SlashCommands/Hostname.cs
+++ b/Shared/MFDLabs.GridCommands/SlashCommands/Hostname.cs
@@ -28,7 +28,7 @@ public async Task Invoke(SocketSlashCommand command)
                     $"The hostname for this instance is: `{SystemGlobal.GetMachineHost()}`\n" +
                     $"The machine ID for this machine is: `{SystemGlobal.GetMachineId()}`\n" +
                     $"The IP address for this machine is: `{NetworkingGlobal.GetLocalIp()}`\n" +
-                    "Please paste this into the `Host Name` field in grid-bot-support templates to that the internal team can easily identify this instance."
+                    "Please paste this into the `Host Name` field in grid-bot-support templates so that the internal team can easily identify this instance."
                 );
             }
         }
diff --git a/Shared/MFDLabs.GridCommands/SlashCommands/LogFile.cs b/Shared/MFDLabs.GridCommands/SlashCommands/LogFile.cs
index d6e12cb..5c014ca 100644
--- a/Shared/MFDLabs.GridCommands/SlashCommands/LogFile.cs
+++ b/Shared/MFDLabs.GridCommands/SlashCommands/LogFile.cs
@@ -23,7 +23,10 @@ public async Task Invoke(SocketSlashCommand command)
         {
             using (await command.DeferEphemeralAsync())
             {
-                await command.RespondEphemeralAsync($"The log file name for this instance is: `{SystemLogger.FileName}`\nPlease paste this into the `Log File Name` field in grid-bot-support templates to that the internal team can easily identify this instance's log files.");
+                await command.RespondEphemeralAsync(
+                    $"The log file name for this instance is: `{SystemLogger.FileName}`\n" + 
+                    "Please paste this into the `Log File Name` field in grid-bot-support templates so that the internal team can easily identify this instance's log files."
+                );
             }
         }
     }
```

**Comment by mfdlabs-ops** [comment]
@AnthonyEagle1776 this PR has been pulled, associated deployments:

Staging: `2022.03.13-23.29.45_fix_inconsistent-spelling-in-commands_76f30e7`
Integration: `2022.03.13-23.32.14_master_96a4f6f`
